frontend/src/app/testQueue/addToQueue/AddToQueueSearch-patient-search.test.tsx

```diff
@@ -55,7 +56,9 @@ const mocks = [
   {
     request: {
       query: QUERY_PATIENT,
+      includeArchived: false,
```

Comment:
is this supposed to be in this part of the request? I'm surprised this isn't causing something to fail. are we using this mock? 🧐